desktop/ui/src/main/java/org/datacleaner/cli/CliRunner.java

```diff
@@ -70,7 +73,6 @@
  * Contains the execution logic to run a job from the command line.
  */
 public final class CliRunner implements Closeable {
-
```

Comment:
Hehe, I don't care that much, it just looks weird to me, like an empty line after other scope operators (which I guess isn't really comparable, but eyes still reacts the same). :)